tgstation.dme

```diff
@@ -6,6 +6,73 @@
 
 // BEGIN_FILE_DIR
 #define FILE_DIR .
+#define FILE_DIR "code"

```

Comment:
dont commit ```FILE_DIR```

Dreammaker --> Help --> Preferences for tgstation --> Untick "automatically define FILE_DIR"

then recompile and recommit.